sources/engine/Stride.Engine/Animations/AnimationBlender.cs

```diff
@@ -294,13 +294,13 @@ public static unsafe void Blend(CoreAnimationOperation blendOperation, float ble
         /// </summary>
         /// <param name="animationOperations">The animation operations to perform.</param>
         /// <param name="result">The optional result (if not null, it expects the final stack to end up with this element).</param>
-        public void Compute(FastList<AnimationOperation> animationOperations, ref AnimationClipResult result)
+        public void Compute(List<AnimationOperation> animationOperations, ref AnimationClipResult result)
```

Comment:
Wondering if in API like this one, the parameter could be `Span<T>`  or `ReadOnlySpan<T>`  instead?